latest version broken due to wrong import

The latest version ov Svelvet 8.0.5 could not be used as a wrong import was introduced within beb4a3c319e7ea430b800f83cddf8339335f3dde.


```
 ✘ [ERROR] Could not resolve "../../../routes/+page.svelte"

    node_modules/svelvet/dist/components/Anchor/Anchor.svelte:44:17:
      44 │ import Page from "../../../routes/+page.svelte";
         ╵                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reference:
https://github.com/open-source-labs/Svelvet/blob/beb4a3c319e7ea430b800f83cddf8339335f3dde/src/lib/components/Anchor/Anchor.svelte#L14

--- a/src/lib/components/Anchor/Anchor.svelte
+++ b/src/lib/components/Anchor/Anchor.svelte
@@ -11,7 +11,6 @@
 	import type { InputType, NodeKey, OutputStore, InputStore, ConnectingFrom } from '$lib/types';
 	import type { ComponentType } from 'svelte';
 	import type { Writable, Readable } from 'svelte/store';
-	import Page from '../../../routes/+page.svelte';
 
 	let animationFrameId: number;
 

